p2panda-rs/src/atomic/entry_signed.rs

```diff
@@ -57,11 +58,12 @@ impl EntrySigned {
     }
 }
 
-impl TryFrom<&EntrySigned> for BambooEntry<&[u8], &[u8]> {
-    type Error = anyhow::Error;
-
-    fn try_from(signed: &EntrySigned) -> Result<Self> {
+impl<'a> From<&'a EntrySigned> for BambooEntry<&'a [u8], &'a [u8]> {
+    fn from(_signed_entry: &'a EntrySigned) -> Self {
         todo!();
+        // let test = signed_entry.clone();
+        // let bytes = test.to_bytes().as_slice();
+        // TryInto::<BambooEntry<&'a[u8], &'a[u8]>>::try_into(bytes).unwrap()
     }
 }
```

Comment:
I looked a little bit into this (with a fresher brain :dog: @cafca and @sophiiistika) and think we have to take another approach here: `bytes` is owned inside the scope of the `from` function and can never be moved out through `BambooEntry` as it is only passed on as a reference.

We can't change that as the bamboo-rs crate only allows references: https://github.com/pietgeursen/bamboo-rs/blob/master/bamboo-rs-core/src/entry/mod.rs#L75

As an alternative we can look into this trait though, assuming that `ArrayVec` allows ownership over the bytes: https://github.com/pietgeursen/bamboo-rs/blob/master/bamboo-rs-core/src/entry/mod.rs#L83

Also `into_owned` might be interesting: https://github.com/pietgeursen/bamboo-rs/blob/master/bamboo-rs-core/src/entry/mod.rs#L101.